app/src/main/res/menu/library_owned_book_item_menu.xml

```diff
@@ -4,7 +4,4 @@
     <item
         android:id="@+id/library_owned_book_menu_delete"
         android:title="@string/book_item_menu_delete"/>
-    <item
-        android:id="@+id/library_owned_book_menu_view_requests"
-        android:title="@string/book_item_menu_view_requests"/>
```

Comment:
deleted the button in popup since we have a new button, this button was visible for all owner books regardless of book status